package.json

```diff
@@ -8,7 +8,7 @@
     "url": "git+https://github.com/apollostack/subscriptions-transport-ws.git"
   },
   "dependencies": {
-    "@types/node": "^6.0.38",
```

Comment:
@DxCx moved the node typings here to solve a problem we had with the release, but I can't quite remember what it was. Are you sure we don't need them any more (it seems like a change unrelated to the PR)